assets/merchant/JRK.json

```diff
@@ -0,0 +1,21 @@
+{
+    "metadata": {
+        "label": "jrk",
+        "name": "Jrk",
+        "category": "merchant",
```

Comment:
I am afraid we can't consider it as a merchant. In general we are not labelling community token, but for tokens with significant level of onchain activity we can consider __other__ category